olp-cpp-sdk-authentication/include/olp/authentication/AuthenticationCredentials.h

```diff
@@ -59,6 +60,39 @@ class AUTHENTICATION_API AuthenticationCredentials {
    */
   const std::string& GetSecret() const;
 
+  /*
+   * @brief Reads credentials to access OLP from an input stream that is
```

Comment:
```suggestion
   * @brief Reads your access credentials from an input stream that is
```

Reply:
Please check the latest revision (e3a90c4) that was pushed 2 hours ago, instead of penultimate.
You will find that this comment is addressed already.

Reply:
Checked: `* @brief Reads credentials to access OLP from an input stream that is`
My comment wasn't addressed.

Reply:
You're right. Sorry, my bad.
Done.